.circleci/config.yml

```diff
@@ -153,9 +153,12 @@ executors:
       - image : prestocpp/velox-avx-circleci:kpai-20221017
     resource_class: 2xlarge
     environment:
+        fmt_SOURCE: BUNDLED
         CC:  /opt/rh/gcc-toolset-9/root/bin/gcc
         CXX: /opt/rh/gcc-toolset-9/root/bin/g++
   check:
+    environment:
+        fmt_SOURCE: BUNDLED
```

Comment:
revert before merge

Reply:
Nice way of testing it! Should we also publish the new docker image or leave it with this environment variable?

Reply:
Thanks! We could add the system deps back into the containers and use the envvar to force either or, that way we are more flexible for testing!

We also might want to introduce a `VELOX_DEPENDENCY_SOURCE` envvar that allows for setting this for all moved deps? @kgpai what do you think?

Reply:
Why not have 2 containers, one with dependencies installed and one without. We can use the installed container to run for some of the longer ci jobs (linux-adapters etc). 
Adding VELOX_DEPENDENCY_SOURCE seems like a good idea to me.

Reply:
My concern was that currently we have dependencies installed in /usr/local leaking into our builds and having these dependencies being gated via the flag might not be properly testing things out. However thinking about it, this might also allow us to test if installing newer dependencies doesnt cause the older dependencies to leak in.